lib/vtls/nss.c

```diff
@@ -2374,6 +2374,9 @@ static CURLcode Curl_nss_md5sum(unsigned char *tmp, /* input */
   PK11Context *MD5pw = PK11_CreateDigestContext(SEC_OID_MD5);
   unsigned int MD5out;
 
+  if(!MD5pw)
+    return CURLE_OUT_OF_MEMORY;
```

Comment:
Technically that's actually not what's happening, though I'm not sure we have a more suitable error code. Maybe we should add a comment stating that the algorithm may have been unsupported by libnss?

Reply:
Yeah, I picked this out of laziness. Perhaps `CURLE_NOT_BUILT_IN` is more suitable?

Reply:
I'm just a little hesitant to add a "algorithm in TLS library wasn't available" error code but maybe we should...

Reply:
I'm leanings towards that we should, but perhaps more generic with something like "unsupported algorithm" where the context will imply the "which" and "where"

Reply:
I'm not sure the context is necessary obvious to a libcurl user but it could still be a step forward from what we have now. Still, as we're now less than 48 hours from release I'll stick to `CURLE_NOT_BUILT_IN` for now, and then we come back to this subject after the release.